packages/functions/src/cache/in-memory-cache.ts

```diff
@@ -0,0 +1,67 @@
+import { FunctionCache } from './types';
+
+interface CacheEntry {
+  value: unknown;
+  lastModified: number;
+  tags?: string[];
```

Comment:
suggestion: Make this non-optional, or use `Set<string>`. It would make some logic simpler:

```suggestion
  tags: string[];
```

Reply:
refactored